org/views.py

```diff
@@ -137,4 +137,44 @@ def Permission_Set_List(request,org_id):
             }
             response_object.append(value)
         return Response(response_object, status.HTTP_200_OK)
-    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
\ No newline at end of file
+    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
+
+
+
+@swagger_auto_schema(
+    operation_id="edit_org",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='PUT',
+    request_body = EditOrgSerializer,
+    responses={
+        '200': set_example(responses.update_org_200),
+        '400': set_example(responses.org_not_present_400),
+        '401': set_example(responses.user_unauthorized_401),
+        '403': set_example(responses.admin_access_403),
+    }
+)
+@api_view(['PUT'])
+@permission_classes([IsAuthenticated])
+def EditOrg(request,org_id):
+    org_count = Org.objects.filter(pk=org_id).count()
+    if org_count>0:
+        org = Org.objects.get(pk=org_id)
+        user = request.user
+        if Member.objects.filter(user=user,org=org).count()>0:
+            isadmin = Member.objects.get(user=user,org=org).permission_set.perm_obj.permissions_to_integer()
+            #Checking if the isadmin is odd or even, if odd then the IS_ADMIN permission is enabled for the user
```

Comment:
Since we've gotten rid of the PermissionSet model, I suppose you would need to pull from upstream and `Member.objects.get(user=user,org=org).permission_set.perm_obj.permissions_to_integer()` needs to change to `Member.objects.get(user=user,org=org).group.perm_obj.permissions_to_integer()`